CMake/SlicerCPack.cmake

```diff
@@ -390,6 +390,12 @@ if(CPACK_GENERATOR STREQUAL "NSIS")
   set(PACKAGE_APPLICATION_NAME "${APPLICATION_NAME} ${CPACK_PACKAGE_VERSION}")
   slicer_verbose_set(CPACK_PACKAGE_EXECUTABLES "..\\\\${EXECUTABLE_NAME}" "${PACKAGE_APPLICATION_NAME}")
 
+  if(WIN32)
+    get_property(${app_name}_CPACK_NSIS_MUI_ICON GLOBAL PROPERTY ${app_name}_WIN_ICON_FILE)
+    slicer_cpack_set("CPACK_NSIS_MUI_ICON")
+    slicer_verbose_set(CPACK_NSIS_INSTALLED_ICON_NAME "${app_name}.exe")
+  endif()
```

Comment:
```suggestion
  # Icon filename (*.ico) used as the main icon for the generated install program.
  get_property(${app_name}_CPACK_NSIS_MUI_ICON GLOBAL PROPERTY ${app_name}_WIN_ICON_FILE)
  slicer_cpack_set("CPACK_NSIS_MUI_ICON")

  # Relative path to the executable that contains the installer icon.
  slicer_verbose_set(CPACK_NSIS_INSTALLED_ICON_NAME "${app_name}.exe")
```

Reply:
Since the code is already enclosed in `if(CPACK_GENERATOR STREQUAL "NSIS")`, it turns out that adding the `if(WIN32)` is not needed at all.

Reply:
If this change looks good, it should then be good for integration.